torntools/scripts/content/faction/ttWarReport.js

```diff
@@ -0,0 +1,53 @@
+warReportLoaded().then(() => {
+	console.log("TT - Faction | War Report");
+
+	displayContainer();
+});
+
+function warReportLoaded() {
+	return requireElement("div.faction-war");
+}
+
+function displayContainer() {
+	let rawHTML = '<div id="ttWarReport"><div class="m-top10 tt-title title-green all-rounded"><div class="title-text">War Report</div><div class="tt-options"></div><i id="ttExportButton" class="tt-title-icon fa fa-table" style=""></i><span id="ttExportButton" class="tt-title title-text" style="padding-left: 2px;padding-right: 12px;font-size: larger;">CSV</span><a id="ttExportLink"></a></div></div>';
```

Comment:
It would be best to use the `content.newContainer` for this. I can push the commit on how I'd do it (without changing too much, as I would do it different in more ways) if you want.